CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ This release introduces the following breaking changes:
 - feat: add glob config provider [#713]
 - feat(build): validate FIPS mode at build time and runtime [#693]
 - feat(ci): add windows builds to dev & pr jobs [#762]
+- feat(packaging/msi): add msi packaging [#852]
```

Comment:
You probably need to add the definition of the link [#852] down below, like this:

```
[#852]: https://github.com/SumoLogic/sumologic-otel-collector/pull/852
```